### ISSUE TITLE
Postgres: Entity with @TableInheritance can't be saved/inserted as Array of entities

### ISSUE THREAD

**Issue by ljobse** [typeorm/typeorm#3600]
**Issue type:**

[ ] question
[x ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x ] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x ] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

Group Entity :
```
@Entity()
@TableInheritance({ column: { type: 'varchar', name: 'discriminator' } })
class Group {
  @Column({ nullable: false })
  public name: string;
}
```

Saving an array:
```
const groups: Group[] = []
const groups.push(await groupRepository.create(groupdto1));
const groups.push(await groupRepository.create(groupdto2));
await groupRepository.save(groups);
// => Throws: Error: QueryFailedError: bind message supplies 28 parameters, but prepared statement "" requires 32
```

It seems whenever I try to save/insert a list of entities that have a @TableInheritance decorator it throws an error that the message did not supply enough parameters.

When I did the same experiment with MySql this works without an issue.

Currently the workaround i'm using is to loop through the list of entities and saving them one by one.

**Comment by galkin** [comment]
@ljobse, I have the same problem. I hope it will be fixed in next version. As temporary fix I use 
`{ chunk: 1 }`

`await groupRepository.save(groups, { chunk: 1 });`

**Comment by Kuzna** [comment]
I have the same problem. I created a small repository showing the problem -> https://github.com/Kuzna/typeorm-bulk-insert-bug

**Comment by Kononnable** [comment]
Duplicate of #2253 which was fixed recently(will be released as `typeorm@0.2.17`).
